crates/bws/Cargo.toml

```diff
@@ -15,23 +15,31 @@ keywords = ["bitwarden", "secrets-manager", "cli"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = { version = "4.3.0", features = ["derive", "env"] }
-tokio = { version = "1.28.2", features = ["rt-multi-thread", "macros"] }
-log = "0.4.18"
+bat = { version = "0.23.0", features = [
+    "regex-onig",
+], default-features = false }
 bitwarden = { path = "../bitwarden", version = "0.2.1", features = ["secrets"] }
+chrono = { version = "0.4.26", features = [
+    "clock",
+    "std",
+], default-features = false }
+clap = { version = "4.3.0", features = ["derive", "env"] }
+color-eyre = "0.6"
+comfy-table = "^7.0.1"
+directories = "5.0.1"
 env_logger = "0.10.0"
-supports-color = "2.0.0"
-thiserror = "1.0.40"
+log = "0.4.18"
 serde = "^1.0.163"
 serde_json = "^1.0.96"
 serde_yaml = "0.9"
-bat = { version = "0.23.0", features = ["regex-onig"], default-features = false }
-directories = "5.0.1"
-color-eyre = "0.6"
+supports-color = "2.0.0"
+thiserror = "1.0.40"
+tokio = { version = "1.28.2", features = ["rt-multi-thread", "macros"] }
 toml = "0.7.4"
-comfy-table = "^7.0.1"
-chrono = { version = "0.4.26", features = ["clock", "std"], default-features = false }
 uuid = { version = "^1.3.3", features = ["serde"] }
 
 [dev-dependencies]
 tempfile = "3.5.0"
+
+[target.'cfg(target_os = "linux")'.dependencies]
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Statically linking this probably activates more of the [OpenSSL license ](https://www.openssl.org/source/license-openssl-ssleay.txt) requirements -- namely, that you have to be able to get at the copyright from the binary and any marketing we do needs to credit OpenSSL

Reply:
I think so, we could look into rusttls which gets rid of the openssl dependency.

Reply:
OpenSSL 3 is also licensed under Apache so we should be fine.

Reply:
yes, I had 1.1.1 in my head